bazel/ray_deps_setup.bzl

```diff
@@ -175,8 +175,9 @@ def ray_deps_setup():
     )
 
     auto_http_archive(
-        url = "https://github.com/google/googletest/archive/refs/tags/v1.13.0.tar.gz",
-        sha256 = "ad7fdba11ea011c1d925b3289cf4af2c66a352e18d4c7264392fead75e919363",
+        name = "com_google_googletest",
```

Comment:
we previously did not have this, and as result, never actually used the googletest package declared here.